### DISCUSSION TITLE
How are packages installed with `--HEAD` kept up-to-date?

### DISCUSSION THREAD

**Discussion by penyuan** [Homebrew/discussions#2665] [Everyday usage] [answered]
Hello, 

I've got some packages, such as `ffmpeg`, installed with the `--HEAD` option. How does Homebrew keep packages like this updated? Is it the same as any other package when updating with `brew update && brew upgrade`?

**Comment by SMillerDev** [comment]
No, AFAIK if you want to install from the latest HEAD you'll have to remove the old version and install again with `--head`

**Reply by penyuan** [reply]
Thanks for the quick response!

**Comment by carlocab** [comment, marked as answer]
`brew` only checks whether `HEAD` installs are outdated when the stable version is updated.

If you want to check before then, you can do `brew outdated --fetch-HEAD` and then `brew upgrade --fetch-HEAD` if there's a newer commit available.

**Reply by penyuan** [reply]
Understood. Appreciate the explanation! Will do.
